File: TrabajoI.C/C#/interfaz/ClaseB.cs
public class ClaseB : IOperable
{
    private int valor;

    public B(int valor)  // Constructor con parámetros
    {
        this.valor = valor;
        Console.WriteLine($"Constructor con parámetros en B: {valor}");
    }

    public void Operar()
    {
        Console.WriteLine("Operando en B");
    }
}

File: TrabajoI.C/JS/interfaz/ClaseB.js
class ClaseB {
    constructor(valor) {
        this.valor = valor;
        console.log(`Constructor con parámetros en B → ${valor}`);

        implementarInterfaz(this, InterfazOperable);
    }

    operar() {
        console.log("Operando en B");
    }
}